sites/all/libraries/CAS/CAS.php

```diff
@@ -20,7 +20,7 @@
  *
  *
  * Interface class of the phpCAS library
- * PHP Version 5
+ * PHP Version 7
```

Comment:
https://github.com/apereo/phpCAS download the most recent release and copy it into place.

Reply:
Thanks!  I added these procedures to doc/phpCAS.md in this PR.